src/foundry/foundry.js/pixi/containers/canvasLayers/placeablesLayers/tokenLayer.d.ts

```diff
@@ -1,157 +1,150 @@
+     *  canDragCreate: false,
+     *  controllableObjects: true,
+     *  rotatableObjects: true,
+     *  zIndex: 100
+     * })
+     * ```
+     */
+    static get layerOptions(): TokenLayer.LayerOptions;
+
+    /**
+     * @override
+     */
+    static documentName: 'Token';
+
+    /** @override */
+    get gridPrecision(): 1;
+
+    /**
+     * Token objects on this layer utilize the TokenHUD
+     */
+    get hud(): TokenHUD;
+
+    /**
+     * An Array of tokens which belong to actors which are owned
+     */
+    get ownedTokens(): ReturnType<this['placeables']['filter']>;
+
+    /** @override */
+    tearDown(): Promise<this>;
+
+    /** @override */
+    activate(): this;
+
+    /** @override */
+    deactivate(): this;
+
+    /**
+     * @override
+     * @param options - (default: `{}`)
+     */
+    selectObjects(options?: SelectOptions): boolean;
+
+    /**
+     * Target all Token instances which fall within a coordinate rectangle.
+     *
+     * @param x             - The top-left x-coordinate of the selection rectangle
+     * @param y             - The top-left y-coordinate of the selection rectangle
+     * @param width         - The width of the selection rectangle
+     * @param height        - The height of the selection rectangle
+     * @param releaseOthers - Whether or not to release other targeted tokens
+     *                        (default: `true`)
+     * @returns The number of Token instances which were targeted.
+     */
+    targetObjects(
+      { x, y, width, height }: { x: number; y: number; width: number; height: number },
+      { releaseOthers }?: { releaseOthers?: boolean }
+    ): number;
+
+    /**
+     * Cycle the controlled token by rotating through the list of Owned Tokens that are available within the Scene
+     * Tokens are currently sorted in order of their TokenID
+     *
+     * @param forwards - Which direction to cycle. A truthy value cycles forward, while a false value cycles backwards.
+     * @param reset    - Restart the cycle order back at the beginning?
+     * @returns The Token object which was cycled to, or null
+     */
+    cycleTokens(forwards: boolean, reset: boolean): Token | null;
+
+    /**
+     * Add or remove the set of currently controlled Tokens from the active combat encounter
+     * @param  state  - The desired combat state which determines if each Token is added (true) or removed (false)
+     *                  (default: `true`)
+     * @param  combat - A Combat encounter from which to add or remove the Token
+     *                  (default: `null`)
+     * @param  token  - A specific Token which is the origin of the group toggle request
+     *                  (default: `null`)
+     * @returns The updated Combat encounter
+     * @remarks Returns the created Combatants or undefined if there is no combat and one can't be created
+     *          or the combat if the user is no GM and wants to remove combatants.
+     */
+    toggleCombat(
```

Comment:
Use configured classes instead.

Maybe worth creating an issue at foundry about the incorrect return type documentation

Reply:
https://gitlab.com/foundrynet/foundryvtt/-/issues/5590